Mathlib/CategoryTheory/Adjunction/Mates.lean

```diff
@@ -108,13 +111,58 @@ def transferNatTrans : (G ⋙ L₂ ⟶ L₁ ⋙ H) ≃ (R₁ ⋙ G ⟶ H ⋙ R
       h.naturality, -Functor.map_comp, ← Functor.map_comp_assoc G, R₂.map_comp]
 #align category_theory.transfer_nat_trans CategoryTheory.transferNatTrans
 
+-- ER: A new construction with a new name.
```

Comment:
It's probably not worth having the construction twice, so we should pick one and use it everywhere (especially since you've already showed they're equal). It would be okay switching to yours, but that would mean you'd need to use it everywhere, which could mean changing existing proofs is a bit tricky

Reply:
Yes if I have your approval, I was planning to do this.